Services/Calendar/templates/default/tpl.day_view.html

```diff
@@ -60,4 +60,9 @@
 		<!-- END time_row -->
 	</tbody>
 </table>
-
+<script>
+    $('.calfullcontent').each(function(){
+        var fs = parseInt($(this).parent().height());
+        $(this).css('height',fs);
+    });
+</script>
```

Comment:
However, why is this inline. Please only do not mix js into templates! Note that this is code duplication from the week view. This is exactly why we should not put js code inline.